src/plugins/typescript.ts

```diff
@@ -9,37 +9,40 @@ import tsdoc, { IJsDocTags } from "ts-quick-docs";
 import { Documentalist, IBlock } from "..";
 import { IFile, IPlugin } from "./plugin";
 
-export interface IDocEntry {
+export interface ITsDocEntry {
     documentation: IBlock;
     fileName: string;
     name: string;
     tags: IJsDocTags;
     type: string;
 }
 
-export interface IPropertyEntry extends IDocEntry {
+export interface ITsPropertyEntry extends ITsDocEntry {
     optional?: boolean;
 }
 
-export interface IInterfaceEntry extends IDocEntry {
+export interface ITsInterfaceEntry extends ITsDocEntry {
     extends?: string[];
-    properties: IPropertyEntry[];
+    properties: ITsPropertyEntry[];
 }
 
-export interface ITypescriptPluginData {
-    ts: IInterfaceEntry[];
+export interface ITsPluginData {
```

Comment:
I think all these sub interfaces are fine with the shortened "Ts" name, but this one in particular should match the name of the plugin so `ITypescriptPluginData`